lib/gollum/app.rb

```diff
@@ -111,7 +116,37 @@ class App < Sinatra::Base
     end
 
     namespace '/gollum' do
-      
+      get '/feed/' do
+        url = env['REQUEST_URI'] || "http#{env['HTTPS'] == 'off' ? nil : 's'}://#{env['SERVER_NAME']}#{env['PATH_INFO']}" # Annoyingly, env['REQUEST_URI'] is not available in the tests. In case it sometimes proves to be unavailable in production, fall back to joining together the various parts of the url.
+        url = url.match(/^(.*)gollum\/feed\/$/)[1]
+        latest_changes = "#{url}gollum/latest_changes"
+        changes = wiki_new.latest_changes(::Gollum::Page.log_pagination_options(per_page: 10, page_num: 0))
+        rss = RSS::Maker.make('2.0') do |maker|
+          maker.channel.author = 'Gollum Wiki'
+          maker.channel.updated = changes.first.authored_date
+          maker.channel.title = "#{@wiki_title} Latest Changes"
+          maker.channel.description = "Latest Changes in #{@wiki_title}"
+          maker.channel.link = latest_changes
+          changes.each do |change|
+            maker.items.new_item do |item|
+              item.link = latest_changes
+              item.title = change.message
+              item.updated = change.authored_date
+              id = change.id
+              files = change.stats.files.map do |files|
+                [files[:old_file], files[:new_file]].compact.map do |file|
+                  f = extract_page_dir(file)
+                  "<li><a href=\"#{url}#{f}/#{id}\">#{f}</a></li>"
```

Comment:
Link to each file's URL at this specific revision.

Reply:
This would work correctly with `page-file-dir` and `base-dir`?

Reply:
Yes:

1. `extract_page_dir` handles the page_file_dir.
2. `base_dir` should be correctly incorporated into `url` on line 121.